athena-db2/src/test/java/com/amazonaws/athena/connectors/db2/integ/Db2IntegTest.java

```
@@ -0,0 +1,224 @@
+        System.setProperty("aws.secretKey", testConfig.getStringItem("secret_key").get());
+        schemaName = testConfig.getStringItem("schema_name").get();
+        tableName1 = testConfig.getStringItem("table_name").get();
+        tableName2 = testConfig.getStringItem("table_name2").get();
+        tableName3 = testConfig.getStringItem("table_name3").get();
+        tableName4 = testConfig.getStringItem("table_name4").get();
+        tableName5 = testConfig.getStringItem("table_name5").get();
+    }
+
+    @Override
+    protected void setUpTableData() {
+        // No-op.
+    }
+
+    @Override
+    protected void setUpStackData(Stack stack) {
+        // No-op.
+    }
+
+    @Override
+    protected void setConnectorEnvironmentVars(final Map environmentVars) {
+        // No-op.
+    }
+
+    @Override
+    protected Optional<PolicyDocument> getConnectorAccessPolicy() {
+        List<String> statementActionsPolicy = new ArrayList<>();
+        statementActionsPolicy.add("*");
+        return Optional.of(PolicyDocument.Builder.create()
+                .statements(ImmutableList.of(PolicyStatement.Builder.create()
+                        .actions(statementActionsPolicy)
+                        .resources(ImmutableList.of("*"))
+                        .effect(Effect.ALLOW)
+                        .build()))
+                .build());
+    }
+
+    @BeforeClass
+    @Override
+    protected void setUp()
+    {
+        try {
+            super.setUp();
+        }
+        catch (Exception e) {
+            //throw e;
+            e.printStackTrace();
+        }
+    }
+
+    public Db2IntegTest()
+    {
+        try{
+            logger.warn("Entered constructor");
+            theApp = new App();
+            lambdaFunctionName = getLambdaFunctionName();
+            lambdaAndSchemaName = "\"lambda:"+lambdaFunctionName+"\".DB2INST2.";
+        }
+        catch (Exception e) {
+            e.printStackTrace();
+        }
```

Same here, why are we just eating all exceptions and only printing them out? Shouldn't this cause a test failure?